tests/mock_crust/churn.rs

```diff
@@ -107,11 +120,35 @@ fn random_churn<R: Rng>(rng: &mut R,
 
         None
     } else {
-        let proxy = rng.gen_range(0, len);
+        let mut proxy = rng.gen_range(0, len);
         let index = rng.gen_range(1, len + 1);
+
+        if nodes.len() > 16 {
```

Comment:
Do we want this constant 16 here? Or was it supposed to be `2 * min_section_size` like somewhere above?